boards/arm/stm32wl5/nucleo-wl55jc/configs/fb/defconfig

```diff
@@ -33,12 +33,9 @@ CONFIG_RAM_START=0x20000000
 CONFIG_RAW_BINARY=y
 CONFIG_SPI=y
 CONFIG_SPI_DRIVER=y
-CONFIG_STM32WL5_LPUART1=y
-CONFIG_STM32WL5_SPI1=y
-CONFIG_STM32WL5_USART1=y
-
```

Comment:
Is it better to change these in this PR? Or separate PR? Or perhaps separate commits of same PR? (I don't have a strong opinion on it. Just asking in case it was missed.)